site/layouts/shortcodes/callout-deprecated-dark-variants.html

```diff
@@ -0,0 +1,9 @@
+{{- /* Outputs message about dark mode component variants being deprecated in v5.3. */ -}}
+
+{{- $component := .Get 0 -}}
+
+<div class="bd-callout bd-callout-warning">
+  <p>
+    <strong>Heads up!</strong> Dark variants for components were deprecated in v5.3.3 with the introduction of color modes. Instead of adding <code>.{{ $component }}-dark</code>, set <code>data-bs-theme="dark"</code> on the root element, a parent wrapper, or the component itself.
```

Comment:
Could be done afterwards but should we try to use this over the whole documentation instead of having multiple warning callouts everywhere ?

```suggestion
    <strong>Heads up!</strong> Dark variants for components were deprecated in v5.3.3 with the introduction of color modes.<br>Add <code>data-bs-theme="dark"</code> to the <code>.{{ $component }}</code> or any ancestor element to enable a component-specific color mode. <a href="/docs/customize/color-modes">Learn more about our color modes</a>.
```